crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
+
+The extension allows you to test this service for free, even if you don't connect the extension to a
+database. After a while, the extension asks you to buy credits to keep using this additional service.
```

Comment:
Might be better to say "The extension allows you to test this service for free without connecting the extension to a database. After a while, the extension asks you to buy credits to continue using this service.

Reply:
To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

Reply:
> Might be better to say "The extension allows you to test this service for free without connecting the extension to a database. After a while, the extension asks you to buy credits to continue using this service.

thanks, but it is actually two different things:
you can try it without connecting
you can try it, even when connected
and you can connect and buy

I'll see how to make this clearer. Thanks!